test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
@cpennington: you change `TamperEvidentFile.write` from (implicitly) bytes to explicit Unicode strings when you made the code run on both 2 and 3.  That now seems like a bad choice of signature: file writing seems like a bytes-oriented operation to me.  But in any case, it should definitely be one or the other.